src/app/lib/config/services/persian.js

```diff
@@ -386,7 +386,7 @@ export const service = {
       },
       {
         title: 'دانش',
```

Comment:
The title for this persian topic page is 'علمی' rather than 'دانش'
```suggestion
        title: 'علمی',
```

Reply:
hi Sareh, I asked that same question and they said it was fine.

Reply:
Okay thanks, Jordi!

Reply:
Thanks for sorting out the Chromatic stuff.